composer.json

```diff
@@ -116,43 +115,17 @@
         "[web-root]/web.config": false
       }
     },
+    "drupal-core-vendor-hardening": {
+      "drupal/module_name": ["tests", "src/Tests"]
+    },
+    "create-required-files": "SkilldDrupal\\composer\\ScriptHandler::createRequiredFiles",
```

Comment:
This is duplicate from https://github.com/skilld-labs/skilld-docker-container/pull/243/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R78